docs/site/index/downloads.md

```diff
@@ -6,100 +6,102 @@ title: CLDR Releases/Downloads
+| 1.4 | 2006-07-17 | v1.4 | [CLDR1.4](https://unicode.org/Public/cldr/1.4.0/) |   | [LDML1.4](https://www.unicode.org/reports/tr35/tr35-6.html) | [~~Δ1.4~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.4) | release-1-4 |   | n/a |
+| 1.3 | 2005-06-02 | v1.3 | [CLDR1.3](https://unicode.org/Public/cldr/1.3.0/) |   | [LDML1.3](https://www.unicode.org/reports/tr35/tr35-5.html) | [~~Δ1.3~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.3) | release-1-3 |   | n/a |
+| 1.2 | 2004-11-04 | v1.2 | [CLDR1.2](https://unicode.org/Public/cldr/1.2.0/) |   | [LDML1.2](https://www.unicode.org/reports/tr35/tr35-4.html) | [~~Δ1.2~~](https://unicode.org/cldr/trac/query?status=closed&col=id&col=summary&milestone=1.2) | release-1-2 |   | n/a |
+| 1.1.1 | 2004-07-29 | n/a | [CLDR1.1.1](https://unicode.org/Public/cldr/1.1.1/) |   | [LDML1.1](https://www.unicode.org/reports/tr35/tr35-2.html) | n/a | release-1-1-1 |   | n/a |
+| 1.1 | 2004-06-08 | n/a | [CLDR1.1](https://unicode.org/Public/cldr/1.1.0/) |   | [LDML1.1](https://www.unicode.org/reports/tr35/tr35-2.html) | n/a | release-1-1 |  | n/a |
+| 1.0 | 2003-12-19 |   |    |see [CLDR 1.0 Release](https://unicode.org/cldr/repository_access.html#CLDR_1.0_Release)|  |  |release-1-0| | n/a |
+
+### Key to Header
+
+| Item | Description|
+|:----:|:-----------|
+| No. | CLDR Version Number. |
+| Date | CLDR Version Release Date. |
+| Rel. Note | A general description of the contents of the release, relevant notes about the release including migration warnings. |
+| Data | A set of zip files containing the contents of the release (the files are complete in themselves, and do not require files from earlier releases -- for the structure of the zip file, see [Repository Organization](/index/downloads#Repository_Organization)). |
 | Charts | A set of charts showing some of the data in the release. |
-| Spec | The version of [UTS #35: LDML](https://www.unicode.org/reports/tr35/) that corresponds to the release |
-| Delta Tickets | A list of all the tickets (fixes and features) in the release, which be used to get the precise corresponding file changes (Some earlier releases are not available in this way due to the JIRA migration, but all of the information is available in GitHub or JIRA.) |
-| Github Tag | The files in the release, accessible via via [Repository Access](/index/downloads#latest_draft_version). For more details see  CLDR Releases (Downloads) |
-| Delta DTDs | A link pointing to a charts of changes in the DTDs over time. |
-
+| Spec | The version of [UTS #35: LDML](https://www.unicode.org/reports/tr35/) that corresponds to the release. |
+| Delta Tickets | A list of all the tickets (fixes and features) in the release, which can be used to get the precise corresponding file changes (Some earlier releases are not available in this way due to the JIRA migration, but all of the information is available in GitHub or JIRA.) |
```

Comment:
Need a period after "changes" and before the parenthesis. (I know I'm _really_ nit-picking now...)